tests/base.py

```diff
@@ -47,6 +49,9 @@
 )
 test_app.testing = True
 
+DISCONNECTED_SIGNALS = {
+    contributor_added: [notify_added_contributor]
+}
```

Comment:
Also, it would be useful to have a brief comment explaining why we disconnect `notify_added_contributor`.